client/src/data/constants.tsx

```diff
@@ -114,4 +115,9 @@ export const SCORE_BOUNDARY_LOW = 0.0;
 export const SCORE_BOUNDARY_THRESHOLD = 0.6;
 export const SCORE_BOUNDARY_PRIORITIZED = 0.75;
 
+
+// Explore the Tool page Map Legend
+export const PRIORITIZED_COMMUNITY = 'Prioritized community';
+export const THRESHOLD_COMMUNITY = 'Threshold community';
```

Comment:
Shouldn't these be localized as well?

Reply:
maybe?

The thinking was that both constants, i.e.,  `*_COMMUNITY` are being called in two places:

1. Here:
https://github.com/usds/justice40-tool/blob/3e8f12ebd57f25433bb144ac24a288b5aec57989/client/src/components/MapLegend/index.tsx#L62

2: In the tests:
https://github.com/usds/justice40-tool/blob/3e8f12ebd57f25433bb144ac24a288b5aec57989/client/src/components/MapLegend/tests/mapLegend.test.tsx#L17

Localization is occurring in both places. if these constants ever change the localization will change as well because of where they are used maybe?

Also since localization is using hooks (useIntl) they may require to be wrapped in functions.

Reply:
Hm, interesting - I wonder if we can call`defineMessages` in this context, without a hook? I'm thinking something like:
```js
 const messages = defineMessages({
    prioritizedCommunity: {
      id: '...,
      defaultMessage: '...',
      description: '...',
    },
```
then at the calling context:
```
intl.formatMessage(constants.messages.prioritizedCommunity)
```
something like this?

seems constants that are localized are a common use case, I think

Reply:
Totally agreed!